warehouse/ingest-core/src/test/java/datawave/ingest/mapreduce/handler/edge/EdgeHandlerTestUtil.java

```diff
@@ -0,0 +1,124 @@
+package datawave.ingest.mapreduce.handler.edge;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Multimap;
+import datawave.ingest.config.RawRecordContainerImpl;
+import datawave.ingest.data.RawRecordContainer;
+import datawave.ingest.data.config.NormalizedContentInterface;
+import datawave.ingest.mapreduce.handler.ExtendedDataTypeHandler;
+import datawave.ingest.mapreduce.job.BulkIngestKey;
+import datawave.ingest.mapreduce.job.writer.AbstractContextWriter;
+import datawave.ingest.test.StandaloneStatusReporter;
+import datawave.ingest.test.StandaloneTaskAttemptContext;
+import datawave.util.TableName;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Value;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.mapreduce.TaskInputOutputContext;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class EdgeHandlerTestUtil {
+    
+    public static final Text edgeTableName = new Text(TableName.EDGE);
+    public static final String NB = "\u0000";
+    
+    public static Set<String> edgeKeyResults = new HashSet<>();
+    
+    private static Logger log = Logger.getLogger(EdgeHandlerTestUtil.class);
+    
+    public static boolean isDocumentKey(Key k) {
+        return isShardKey(k) && k.getColumnFamily().toString().equals(ExtendedDataTypeHandler.FULL_CONTENT_COLUMN_FAMILY);
+    }
+    
+    public static boolean isShardKey(Key k) {
+        return k.getRow().toString().matches("\\d{8}_\\d+");
+    }
+    
+    public static void processEvent(Multimap<String,NormalizedContentInterface> eventFields, ExtendedDataTypeHandler<Text,BulkIngestKey,Value> edgeHandler,
+                    RawRecordContainer event, int expectedEdgeKeys, boolean printKeysOnlyOnFail, boolean edgeDeleteMode) {
+        
+        Assert.assertNotNull("Event was null.", event);
```

Comment:
I have a preference for static imports for assert methods, but that won't hold up an approval